lib/trailblazer/test/operation/assertions.rb

```diff
@@ -1,16 +1,14 @@
 module Trailblazer::Test::Operation
   module Assertions
-    # @needs params_pass
-    # @needs attributes_pass
-    def assert_pass(operation_class, params, expected_attributes, default_params: params_pass, default_attributes: attrs_pass, &block)
-      input_params        = default_params.merge( params )
-      expected_attributes = default_attributes.merge( expected_attributes )
+    def assert_pass(operation_class, params, expected_attributes, default_params: {}, default_attributes: {}, &block)
+      input_params        = params.merge( default_params ){|key,oldval,newval| oldval }
```

Comment:
Choosing to keep the values in `params` instead of the one from `default_params` in case a key is defined in both hashes